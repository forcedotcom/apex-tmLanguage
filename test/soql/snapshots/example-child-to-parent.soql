SELECT Contact.FirstName, Contact.Account.Name FROM Contact;
SELECT Id, Name, Account.Name FROM Contact WHERE Account.Industry = 'media'
