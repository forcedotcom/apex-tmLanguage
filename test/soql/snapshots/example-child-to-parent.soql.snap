>SELECT Contact.FirstName, Contact.Account.Name FROM Contact;
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^^^^^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                        ^ source.soql punctuation.separator.comma.apex
#                         ^ source.soql
#                          ^^^^^^^^^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                                              ^ source.soql
#                                               ^^^^ source.soql keyword.operator.query.from.apex
#                                                   ^ source.soql
#                                                    ^^^^^^^ source.soql storage.type.apex
#                                                           ^^ source.soql
>SELECT Id, Name, Account.Name FROM Contact WHERE Account.Industry = 'media'
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^ source.soql keyword.query.field.apex
#         ^ source.soql punctuation.separator.comma.apex
#          ^ source.soql
#           ^^^^ source.soql keyword.query.field.apex
#               ^ source.soql punctuation.separator.comma.apex
#                ^ source.soql
#                 ^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                             ^ source.soql
#                              ^^^^ source.soql keyword.operator.query.from.apex
#                                  ^ source.soql
#                                   ^^^^^^^ source.soql storage.type.apex
#                                          ^ source.soql
#                                           ^^^^^ source.soql keyword.operator.query.where.apex
#                                                ^ source.soql
#                                                 ^^^^^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                                                                 ^ source.soql
#                                                                  ^ source.soql keyword.operator.assignment.apex
#                                                                   ^ source.soql
#                                                                    ^ source.soql string.quoted.single.apex punctuation.definition.string.begin.apex
#                                                                     ^^^^^ source.soql string.quoted.single.apex
#                                                                          ^ source.soql string.quoted.single.apex punctuation.definition.string.end.apex
>